swift/vapor/Sources/main.swift

```diff
@@ -0,0 +1,19 @@
+import Vapor
+
+let drop = Droplet(environment: .production)
```

Comment:
Nevermind, without this vapor falls back to default logging, which it shouldn't for this use case.